packs/opentelemetry_collector/templates/opentelemetry_collector.nomad.tpl

```diff
@@ -32,11 +32,17 @@ job [[ template "full_job_name" . ]] {
 
     [[ template "vault_config" . ]]
 
-    task "opentelemetry_collector" {
+    task "otel-collector" {
       driver = "docker"
 
       config {
         image = "[[ $vars.task_config.image ]]:[[ $vars.task_config.version ]]"
+        force_pull = true
```

Comment:
Maybe this should be configurable? Although, in general, I don't think it would be necessary to set it. Did you encounter any problem without setting this?

Reply:
I haven't had any issues with this particular case when I omitted `force_pull = true`. I'm okay to remove it altogether, since it's a moot point. Maybe it only applies more if people set `$vars.task_config.version` to `latest`, which I hope they don't. :) I could always do `force_pull = true` if `$vars.task_config.version == latest`. What do you think?

Reply:
> I could always do force_pull = true if $vars.task_config.version == latest.

This is Nomad's default behaviour 🙂 

> If image's tag is latest or omitted, the image will always be pulled regardless of this setting.
https://www.nomadproject.io/docs/drivers/docker#force_pull

That's why I think we can just remove this config.

Reply:
Makes sense. And done!